src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -101,7 +101,26 @@ <h2 class="h2 title" id="notifications">{% trans "Voorkeuren voor meldingen" %}
         </div>
     </section>
 
-    {% if view.config.selected_categories or view.config.mentors or view.config.my_contacts or view.config.actions or view.config.ssd or view.config.selfdiagnose or view.config.newsletters or view.config.appointments %}
+    {% if view.config.newsletters and form.fields.newsletters.choices %}
+    <section class="tabled tabled--flexible profile-section profile-section--bordered" id="profile-newsletters">
+        <div class="tabled__section">
+            <h2 class="h2 title" id="newsletters">{% trans "Nieuwsbrieven" %}</h2>
+        </div>
+            <form method="POST" id="newsletter-form" class="form" novalidate>
```

Comment:
The top-padding for forms inside a tabled section is too big here but I can change that in an upcoming PR.